### DISCUSSION TITLE
[@types/fs-extra] Upgrade to v11

### DISCUSSION THREAD

**Discussion by dscalzi** [DefinitelyTyped/DefinitelyTyped#63476] [Issues with a @types package] [answered]
v11 adds support for ESM exports.

https://github.com/jprichardson/node-fs-extra/blob/master/CHANGELOG.md

[a comment by an automated account is not shown]

**Comment by dscalzi** [comment]
bump

**Reply by BendingBender** [reply]
Opened a PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63829

**Reply by dscalzi** [reply]
Thanks a ton

**Comment by dscalzi** [comment, marked as answer]
Resolved by @BendingBender
